js/views/modals/moderatorDetails.js

```diff
@@ -55,6 +55,8 @@ export default class extends BaseModal {
         return langData && langData.name || lang;
       });
 
+    const verifiedMod = app.verifiedMods.get(this.model.get('peerID'));
```

Comment:
If this.verifiedModModel already has the model, why switch to getting it a second time here?

Or is that a result of a merge?